modules/sdk-test-connect-disconnect-cs/Lib.cs

```diff
@@ -1,28 +1,28 @@
+namespace SpacetimeDB.Sdk.Test.ConnectDisconnect;
+
 using SpacetimeDB;
 
-static partial class Module
-{
-    [SpacetimeDB.Table(Public = true)]
-    public partial struct Connected
-    {
-        public Identity identity;
-    }
+[SpacetimeDB.Table(Public = true)]
+public partial struct Connected {
```

Comment:
What happened to all these? 😢

Reply:
Dotnet csharpier doesn't seem to work in the module projects

Reply:
used `dotnet format` instead